docs/porting/porting_full_process/built_in_tests.md

```diff
@@ -23,15 +23,15 @@ Read the following page to understand how tests are structured:
 <!--From Jimmy: Don't export tests to Eclipse or at all. It's not supported, AKA not in CI-->
 
 ### Prerequisites
-
+<!-- We should have a link to https://github.com/ARMmbed/greentea or at least command to tell user how to download greentea for this page -->
```

Comment:
If they've installed `mbed-cli`, then they should have greentea installed as well.

Reply:
Here is what my line of thought was, since the title of the page was Testing with greentea framework, I think it would be nice to include a link somewhere which points them to mbed-os greentea documentation. Not a big deal, but just a suggestion :)

Reply:
Ah ok, fine with me!